config/crd/bases/atlas.mongodb.com_atlasdatafederations.yaml

```diff
@@ -111,8 +105,6 @@ spec:
                 required:
                 - name
                 type: object
-              skipRoleValidation:
-                type: boolean
```

Comment:
I think this is a required field, no?

Reply:
not for creation but for updates. I am setting now also `false` for updates.